app/code/Magento/PageCache/Block/FormKeyProvider.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\PageCache\Block;
+
+use Magento\Framework\View\Element\Template;
+use Magento\PageCache\Model\Config;
+
+/**
+ * Adds script to update form key from cookie after script rendering
+ * @api
```

Comment:
Could you remove `@api` tag here?

Reply:
then I have static tests failure

Reply:
Ok... now we have SVC failure because of it...
Anyway any of this failures should be approved